### PR TITLE
Fully refactor projects page to new design with edit and delete

### PR DESCRIPTION
**Pull request by mohe2015**

### CODE REVIEW

**Comment by JuliaSelua** [comment]
Behaviour-wise seems to be working

File: js/src/main/scala/webapp/pages/ProjectsPage.scala [hunk 1]
```diff
@@ -41,99 +40,252 @@ import webapp.services.Page
...
+import kofre.time.VectorClock
 
-private class NewProjectRow {
+private class ProjectRow(existingValue: Option[Synced[Project]], editingValue: Var[Option[Project]]) {
 
-  private val name = Var("")
-  private val maxHours = Var("")
-  private val account = Var("")
+  def updateName(x: String) = {
+    println("hi")
```
Comment (JuliaSelua) [inline js/src/main/scala/webapp/pages/ProjectsPage.scala:48]: some relicts?

File: js/src/main/scala/webapp/pages/ProjectsPage.scala [hunk 2]
```diff
@@ -41,99 +40,252 @@ import webapp.services.Page
...
+                val res = if (p.exists) {
+                  Some(
+                    tr(
+                      data.id := syncedProject.id,
+                      td(p.name),
+                      td(p.maxHours),
+                      td(
+                        {
+                          p.accountName
+                            .reduceOption((a, b) => {
```
Comment (JuliaSelua) [inline js/src/main/scala/webapp/pages/ProjectsPage.scala:159]: this is the "bug fix" for the editing at the same time - atm only implemented for the account, maxTime and name missing

**Review by JuliaSelua — COMMENTED** [review]
Just to mention: this editing conflict is atm only fixed for the accountName.

And btw. what about deleting and editing at the same time?

File: js/src/main/scala/webapp/pages/ProjectsPage.scala (hunk 1 above)
Comment (mohe2015) [reply js/src/main/scala/webapp/pages/ProjectsPage.scala:48]: Yes, I removed them.

**Pull request merged**